CHANGELOG.md

```diff
@@ -44,18 +50,23 @@ All notable changes to this project will be documented in this file. The format
   - Fix repetition of select text on preferences ([#1270](https://github.com/bloom-housing/bloom/pull/1270)) (Emily Jablonski)
   - Fix aplication submission and broken test ([#1270](https://github.com/bloom-housing/bloom/pull/1282)) (Dominik Barcikowski)
   - Fix broken application search in Partners ([#1301](https://github.com/bloom-housing/bloom/pull/1301)) (Dominik Barcikowski)
+    <<<<<<< HEAD
```

Comment:
Guessing this was one we missed?

Reply:
Oh I think this [merge conflict omission](https://github.com/bloom-housing/bloom/blob/master/CHANGELOG.md) is part of their upstream changes. I can fix it here though.